.github/workflows/ci-pipeline.yaml

```diff
@@ -95,13 +107,63 @@ jobs:
           # Container name
           VIB_ENV_CONTAINER: ${{ steps.get-container-metadata.outputs.name  }}
           VIB_ENV_TAG: ${{ steps.get-container-metadata.outputs.tag }}
-  verify-all:
+  verification-summary:
     # Ensure all containers passed the verification
     runs-on: ubuntu-latest
-    name: Verify
-    needs: vib-verify
+    name: Verification Summary
+    needs:
+      - get-containers
+      - vib-verify
     if: ${{ always() }}
     steps:
-      - name: Verify
-        if: ${{ needs.vib-verify.result != 'success' && needs.vib-verify.result != 'skipped'}}
-        run: exit 1
+      - name: Verification Summary
+        run: |
+          if [[ "${{ needs.get-containers.result }}" != "success" ]]; then
+            echo "Verification is required."
+            echo " If you've just created this PR, don't worry about this message. Bitnami Team has to review it and make possible the verification."
+            exit 1
+          else
+            if [[ "${{ needs.get-containers.outputs.result }}" == "skip" ]]; then
+              echo "It seems these changes don't involve any container"
+            else
+              if [[ "${{ needs.vib-verify.result }}" != "success" ]]; then
+                echo "Verification is required"
+                echo "Please review previous jobs to get more information"
+                exit 1
+              else
+                echo "Well done! Everything looks good. Please wait for the Bitnami Team review."
+              fi
+            fi
+          fi
+  auto-pr-review:
+    runs-on: ubuntu-latest
+    name: Reviewal for automated PRs
+    if: |
+      always() &&
+      github.actor == 'bitnami-bot'
```

Comment:
If I'm not mistaken, this will launch the job for every labeled and synchronized event triggered by a PR created by `bitnami-bot`. 

I don't think it is necessary to execute this once the CI has failed once. If an agent needs to check it, I would avoid automerging anything in case the process requires the PR not to be merged immediately. We would also avoid additional comments if the CI keeps failing.

I think we could add another condition so the CI should have been triggered by the labeling of `verify`, though that is not a perfect solution.

Reply:
I am not sure if I got it well. If the PR fails and some user review it, resolving conflicts or adding other labels the action is triggered by that user not bitnami-bot.

Reply:
Maybe I am missing something, but `bitnami-bot` will no longer label the PRs & I don't expect it to push more commits (and trigger synchronized) either, so this condition might be enough.